types/selenium-webdriver/index.d.ts

```diff
@@ -14,7 +15,8 @@ import * as edge from './edge';
 import * as firefox from './firefox';
 import * as ie from './ie';
 import { By, ByHash } from './lib/by';
-import { Command, ICommandName, Name } from './lib/command';
+import * as command from './lib/command';
+import * as http from './http';
```

Comment:
I don't think you will need this import. See comment below about `command.Executor`.